products/devstudio/_common/product.yml

```diff
@@ -7,9 +7,9 @@ youtube_album: https://www.youtube.com/playlist?list=PLf3vm0UK6HKrxJ9OtX2Zmgw3ny
 buzz_tags:
   - jbds
 guides:
-  Release_Notes:
-  Install_Red_Hat_JBoss_Developer_Studio:
-  Update_Red_Hat_JBoss_Developer_Studio:
+  Release_Notes: https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_Developer_Studio/9.1-Beta1/html/9.1_Beta1_Release_Notes/index.html
```

Comment:
This is the wrong syntax for hard coding the URLS in. See the example here: https://github.com/jboss-developer/www.jboss.org/blob/master/products/amq/_common/product.yml#L26-L35.

However, it might not be needed, as the URLs are predicted based on the value of `current_version`.

Furthermore, we only display docs for the latest GA release on this page. So you shouldn't be changing it for Beta releases.